runtime/src/protocol.rs

```diff
@@ -470,7 +467,7 @@ impl Protocol {
 
         #[cfg(target_env = "sgx")]
         {
-            if self.rak.quote().is_none() {
+            if self.identity.quote().is_none() {
```

Comment:
Could simplify this whole section to `self.identity.quote().ok_or(ProtocolError::...)?;`. And maybe similar above.